package.json

```diff
@@ -32,10 +32,10 @@
     "@babel/preset-env": "^7.8.7",
     "@babel/preset-react": "^7.8.3",
     "@svgr/rollup": "^4.3.0",
-    "babel-plugin-styled-components": "^1.10.0",
     "enzyme": "^3.11.0",
     "govuk-react": "^0.7.1",
     "jest": "^25.1.0",
+    "jest-styled-components": "^7.0.2",
```

Comment:
What is the reason for this change?

Reply:
This adds style information to the test snapshot, so we can spot style regressions.

Reply:
What about the removal of the babel plugin?